configuration/configuration.go

```diff
@@ -329,7 +329,7 @@ type Health struct {
 type v0_1Configuration Configuration
 
 // UnmarshalYAML implements the yaml.Unmarshaler interface
-// Unmarshals a string of the form X.Y into a Version, validating that X and Y can represent uints
```

Comment:
This is not a misspelling, `uints` is referring to unsigned integers. Could be rephrased to make it clearer.